src/frontend.md

```diff
@@ -233,6 +233,8 @@ export function fakeTicket() {}
 
 1. Design components API for common usage rather than for specific one.
 
+1. Prefer S3 distribution of your assets. Put fonts, images, icons to `src/assets`.
```

Comment:
I would add that we can store svg icon as a code if it's not too large. For example, if it's smaller than 3 or 4 kb.

Reply:
I wonder if we really need this way for UI Kit. I believe the only difference is assets blinking on loading.

Reply:
the reason to store static files is reducing bundle size and reusing cache, but there is drawback – blinking on loading

so in case of 3-4 kb icons we do not win too much here, but get assets blinking, so what's the reason for assets async loading in this case?

Reply:
Agree, but we need to prevent somehow bloating of the bundle size. if we would have 100 icons it will lead to additional 400kb. 

I believe we should migrate client-ui-kit to use separate exports as in agent-ui-kit. then we may allow using inlined icons. wdyt?

Also usage of S3 works well with caching. 
I haven't encountered situation yet when it was problematic to use.
So with problem assets blinking usually is hidden.

Reply:
jfyi: https://github.com/OsomePteLtd/client-ui-kit/pull/513

you can help to finish that PR :)

Reply:
let's approve this PR then? :D